components/gitpod-protocol/src/protocol.ts

```diff
@@ -293,11 +293,24 @@ export namespace EnvVar {
             return res;
         }
 
+        // Returns a host value, which can include port, if token also has length.
+        const getHost = (parts: string[]): string => {
+            if (parts.length === 2 && parts[0] && parts[1]) {
+                return parts[0];
+            } else if (parts.length === 3 && parts[0] && parts[1] && parts[2]) {
+                return `${parts[0]}:${parts[1]}`;
+            }
+            return "";
```

Comment:
In TS, `undefined´ takes the role of "" in Go. Why not stick to that convention.

Reply:
Fixed in d444551a4b93038190b9d035a5c3eea3545bfb55